msal/application.py

```diff
@@ -151,6 +151,14 @@ def __init__(
             "The provided signature value did not match the expected signature value",
             you may try use only the leaf cert (in PEM/str format) instead.
 
+            *Added in version 1.13.0*:
+            It can also be a completly pre-signed assertion that you've assembled yourself.
+            Simply pass a container containing only the key "client_assertion", like this:
```

Comment:
```suggestion
            Simply pass a container containing only the key "client_assertion", like this::
```
The [restructuredtext needs a double-colon](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks) here.

By the way, this suggestion can be accepted by one-click from the Github web interface.